src/test-support/mocks/src/zoneingresses+insights/_.ts

```diff
@@ -0,0 +1,89 @@
+import type { EndpointDependencies, MockResponder } from '@/test-support'
+export default (_deps: EndpointDependencies): MockResponder => (req) => {
+  const params = req.params
+
+  return {
+    headers: {},
+    body: {
+      type: 'ZoneIngressOverview',
+      name: params.name,
+      creationTime: '2021-07-13T08:40:59Z',
+      modificationTime: '2021-07-13T08:40:59Z',
+      zoneIngress: {
+        zone: 'zone-1',
+        networking: {
+          address: '10.60.0.16',
+          advertisedAddress: '35.239.169.183',
+          port: 10001,
+          advertisedPort: 10001,
+        },
+        availableServices: [
+          {
+            tags: {
+              app: 'demo-app',
+              'kuma.io/protocol': 'http',
+              'kuma.io/service': 'demo-app_kuma-demo_svc_5000',
+              'kuma.io/zone': 'zone-1',
+              'pod-template-hash': '5845d6447b',
+            },
+            instances: 1,
+            mesh: 'default',
+          },
+          {
+            tags: {
+              app: 'redis',
+              'kuma.io/protocol': 'tcp',
+              'kuma.io/service': 'redis_kuma-demo_svc_6379',
+              'kuma.io/zone': 'zone-1',
+              'pod-template-hash': '59c9d56fc',
+            },
+            instances: 1,
+            mesh: 'default',
+          },
+        ],
+      },
+      zoneIngressInsight: {
+        subscriptions: [
+          {
+            id: 'cc2743b5-43e0-45b6-be88-956ea91a4aad',
+            controlPlaneInstanceId: 'kuma-control-plane-84f5589874-nmspq-0867',
+            connectTime: '2021-07-13T08:41:04.556796688Z',
+            generation: 409,
+            status: {
+              lastUpdateTime: '2021-07-13T09:03:11.614941842Z',
+              total: {
+                responsesSent: '8',
+                responsesAcknowledged: '9',
+              },
+              cds: {
+                responsesSent: '3',
+                responsesAcknowledged: '3',
+              },
+              eds: {
+                responsesSent: '2',
+                responsesAcknowledged: '3',
+              },
+              lds: {
+                responsesSent: '3',
+                responsesAcknowledged: '3',
+              },
+              rds: {},
+            },
+            version: {
+              kumaDp: {
+                version: '1.2.1',
+                gitTag: '1.2.1',
+                gitCommit: 'e88ec407e669c47d3dc9ef32fcde60e2f31c0c4d',
+                buildDate: '2021-06-30T14:32:58Z',
+              },
+              envoy: {
+                version: '1.18.3',
+                build: '98c1c9e9a40804b93b074badad1cdf284b47d58b/1.18.3/Clean/RELEASE/BoringSSL',
+              },
+            },
+          },
+        ],
+      },
+    },
+  }
+}
```

Comment:
We should try to get used to making these as dynamic as possible when we add them (i.e. use `fake` to make all the values different). That way we get a good coverage of different types of data when developing. Obvs if there isn't time to do this for whatever reason then we can always circle back when there is. For example I would like to go through all the newly added ones at some point and make them way more dynamic. No action here if you don't want to do this now btw.

Reply:
I’ve made some updates to the added mocks to make them more random.

Reply:
ty!